scripts/wdl/README.md

```diff
@@ -6,5 +6,70 @@ https://github.com/broadinstitute/wdl
+- ``case_gatk_acnv_workflow.target_file`` --  same target file (NOT bed format) as was used for the PoN creation
+- ``case_gatk_acnv_workflow.gatk_jar`` --  absolute path to gatk-protected.jar 
+- ``case_gatk_acnv_workflow.PadTargets.mem`` -- (Advanced) Number of GB RAM to use when running PadTargets.  Always use 1, unless your target file is really large
+- ``case_gatk_acnv_workflow.CalculateTargetCoverage.disable_all_read_filters`` -- (Advanced) Disables all read filters when making coverage profile.  This will probably have unexpected results.  Always use "false".
+- ``case_gatk_acnv_workflow.CalculateTargetCoverage.mem`` -- Number of GB RAM to use when running CalculateTargetCoverage.  Most exome files do not need more than 4. 
+- ``case_gatk_acnv_workflow.ref_fasta_dict`` --  Path to reference dict file.  Broad internal: /seq/references/Homo_sapiens_assembly19/v1/Homo_sapiens_assembly19.dict 
+- ``case_gatk_acnv_workflow.CalculateTargetCoverage.keepduplicatereads`` -- (Advanced) Whether coverage calculations should count duplicate reads.  Setting this to "true" has been shown (empirically) to improve performance.  "false" is also supported.
+- ``case_gatk_acnv_workflow.PoN`` --  Path to GATK PoN file
+- ``case_gatk_acnv_workflow.CalculateTargetCoverage.transform `` -- (Advanced) Always "PCOV"
+- ``case_gatk_acnv_workflow.ref_fasta`` -- Path to reference fasta file.  Broad internal: ``/seq/references/Homo_sapiens_assembly19/v1/Homo_sapiens_assembly19.fasta`` 
+- ``case_gatk_acnv_workflow.NormalizeSomaticReadCounts.mem`` -- "4" Number of GB RAM to use when running 
+- ``case_gatk_acnv_workflow.PadTargets.pd`` -- (Advanced) "250" has been shown to improve results.  This parameter is number of bases to pad the target file in ``PadTargets``
+- ``case_gatk_acnv_workflow.disable_sequence_dictionary_validation`` -- (Advanced) "true" 
+- ``case_gatk_acnv_workflow.plots_dir`` -- (Advanced)  path to store produced plots for CopyRatio and ACNV 
+- ``case_gatk_acnv_workflow.call_cnloh_dir`` -- (Advanced)  path to directory to store CNLoH and splits results 
+- ``case_gatk_acnv_workflow.enable_gc_correction`` -- "true" is recommended. true/false option change to false to bypass gc bias correction step 
+- ``case_gatk_acnv_workflow.isWGS`` -- Change to true if running WGS samples.  You should also change ``case_gatk_acnv_workflow.seg_param_undoSplits`` and ``case_gatk_acnv_workflow.seg_param_undoSD`` to reduce (not eliminate) hyper-segmentation
+- ``case_gatk_acnv_workflow.wgsBinSize`` -- "10000" .  Size of bins (in bp) for WGS read counts CLI.  *This must be the same as was used to generate the PoN*.  Ignored if not running WGS.
+
+*For segmenter parameters, see the explanation in the Segmenter tool or page 11 of https://bioconductor.org/packages/release/bioc/manuals/DNAcopy/man/DNAcopy.pdf*
```

Comment:
Segmenter -> PerformSegmentation here and below.  However, the argument descriptions there just refer to the manual, so I would just say "see pgs. 11-12 of https://bioconductor.org/packages/release/bioc/manuals/DNAcopy/man/DNAcopy.pdf"